draft-packages/empty-state/KaizenDraft/EmptyState/EmptyState.tsx

```diff
@@ -94,8 +96,20 @@ export const EmptyState: React.VFC<EmptyStateProps> = ({
       </div>
       <div className={styles.textSide}>
         <div className={styles.textSideInner}>
-          <div className={styles.heading}>{headingText}</div>
-          <div className={styles.description}>{bodyText}</div>
+          <Box mb={1}>
+            <Heading
+              variant="heading-3"
+              classNameOverride={styles.heading}
+              tag="div"
```

Comment:
I've left this as a non semantic tag to ensure that the heading level structure of pages that use this component will not be broken

Reply:
not ideal but we could potentially add a prop to make this more accessible by giving them the option to set the tag level on the component - something like `headingTag`?

Reply:
Potentially, but I'd call that out for now. It behaves as it always did using the `div`.